### ISSUE TITLE
Flex-wrap not works

### ISSUE THREAD

**Issue by egdev6** [wkhtmltopdf/wkhtmltopdf#4678]
**wkhtmltopdf version(s) affected**: 0.12.5 (patched)

**OS information**
Ubuntu 19.04

**Description**
Flex-Wrap is ignored and render all the flex's content inline.

**How to reproduce**
```
.week{
   display:-webkit-box;
   display:-ms-flexbox;
   display:flex;
   -ms-flex-wrap: wrap;
   flex-wrap: wrap;
   -webkit-flex-wrap: wrap;
}

.day{
  box-sizing:border-box;
  height:50px;
  border:1px solid black;
  -webkit-box-flex:2;
  -webkit-flex:2;
   flex:2;
   flex-basis:20%;
   -webkit-flex-basis:20%;
}
```

```
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
</head>
<body>
	<div class="week">
		<div class="day">day 1</div>
		<div class="day">day 2</div>
		<div class="day">day 3</div>
		<div class="day">day 4</div>
		<div class="day">day 5</div>
		<div class="day">day 6</div>
		<div class="day">day 7</div>
	</div>
</body>
</html>
```

**Expected behavior**
Expected 2 rows with 6th and 7th day in the second row using 100% width, but get one line with 7 days inline. I think that the conversion ignore -webkit-flex-wrap: wrap;

I would appreciate a possible solution to this problem, thanks.

**Comment by randsu** [comment]
In dotnetcore 3.1 razor page, using `Rotativa.AspNetCore 1.2.0-beta`,  and wkhtmltopdf.exe unpacked from `wkhtmltox-0.12.6-1.msvc2015-win64.exe`, neighter CSS styling `display:flex` nor `display:grid` is working correctly.

**Comment by nhacken** [comment]
Having the same problem with wkhtmltopdf 0.12.5 on macOS.
Tried to style some stuff with `display:grid;` and failed, tried again to arrange it with `display:flex; flex-direction:row;` but my DIVs are displayed as `block`.

**Comment by insanum** [comment]
Same for me on macos with 0.12.6-2. Flex divs no longer renders correctly.

Interestingly, note that it DOES work for me with 0.12.4 and 0.12.5-1 (unfortunately can't use 0.12.5 because of the ToC bug). Hopefully this regression can be quickly addressed.

**Comment by rlam3** [comment]
@egdev6 @nhacken were you able to find a workaround?

**Comment by nhacken** [comment]
> 
> 
> @egdev6 @nhacken were you able to find a workaround?

No, not really a workaround - or better - a lot of work... I'm working anyway with a script to resize some of the content. Now The script must keep track of the different sizes of the content and apply an explicit size to the parent div, this solves most of the problems

**Comment by StevenDeRoover** [comment]
Is there any bugfix/workaround for?  I would like to implement a wrapper as well (in my case having maximum 4 photo's in a "grid" on one page, on the 5th image going to the next page).
